client/blocks/importer/wordpress/content-chooser/index.tsx

```diff
@@ -1,153 +0,0 @@
-import { NextButton, SelectItems } from '@automattic/onboarding';
-import { useI18n } from '@wordpress/react-i18n';
-import clsx from 'clsx';
-import React, { useEffect, useRef, useState } from 'react';
-import illustrationImg from 'calypso/assets/images/onboarding/import-2.svg';
-import ActionCard from 'calypso/components/action-card';
-import FormattedHeader from 'calypso/components/formatted-header';
-import { preventWidows } from 'calypso/lib/formatting';
-import wpcom from 'calypso/lib/wp';
-import { jetpack } from 'calypso/signup/icons';
```

Comment:
This might as well be the last usage of this icon - let's clean up.

Reply:
Removed!